src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/SentenceTerm.java

```diff
@@ -36,7 +36,15 @@
 @NamedEntityGraph(
     name = "sentence-term-with-offender-sentence",
     attributeNodes = {
-        @NamedAttributeNode(value = "offenderSentence")
+        @NamedAttributeNode(value = "offenderSentence", subgraph = "offender-sentence-calc-type"),
```

Comment:
Small PR - this had already been included as part of another PR, but this now reduces the sql for the endpoint to just 1 call